### PR TITLE
HTML-CSS-Popup maksymenko

### PR DESCRIPTION
**Pull request by Meylin-V**
### HTML-CSS-Popup

[Demo](https://meylin-v.github.io/html-css-popup/) |
[Code base](https://github.com/Meylin-V/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Meylin-V** [comment]
I checked common mistakes, it's seems all ok, ready to review

File: submissions/maksymenkonat/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+              <a href="#" class="link">Mail</a>
+            </li>
+            <li class="menu_section">
+              <a href="#" class="link">Images</a>
+            </li>
+            <li class="menu_section">
+              <input type="checkbox" id="menu_checkbox" />
+              <label for="menu_checkbox" class="menu_checkbox">
+                <img src="assets/popup-button.png" alt="popup" />
+              </label>
```
Comment (A-Ostrovnyy) [inline submissions/maksymenkonat/popup/index.html:29]: Fix styles on focus state
image

File: submissions/maksymenkonat/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+              <div class="checkbox-main-menu">
+                <ul class="dropdown_menu">
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/calendar.png" alt="Calendar" />
```
Comment (A-Ostrovnyy) [inline submissions/maksymenkonat/popup/index.html:34]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/maksymenkonat/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                    <a href="#" class="one_section">
+                      <img src="assets/documents.png" alt="Document" />
+                      <p class="text">Document</p>
+                    </a>
+                  </li>
+                </ul>
+                <input type="checkbox" id="more_checkbox" />
+                <label for="more_checkbox" class="menu_checkbox">
+                  <span class="button_more">More</span>
+                </label>
```
Comment (A-Ostrovnyy) [inline submissions/maksymenkonat/popup/index.html:108]: This button is not accessible from the keyboard, please fix it. Also, hide this button when the second part of the popup is open. [Watch video](https://share.vidyard.com/watch/AjhUWq8C2dd3PzNAy2eynq?)

File: submissions/maksymenkonat/popup/style.css [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+.content {
+  min-height: 100%;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+}
+
+ nav {
```
Comment (A-Ostrovnyy) [inline submissions/maksymenkonat/popup/style.css:30]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response!

**Pull request closed without merging**